src/applications/login/components/VerifyPage.jsx

```diff
@@ -30,23 +30,27 @@ function handleClick(version = API_VERSION) {
 
 export default function VerifyPage() {
   return (
-    <div className="row">
-      <va-alert visible status="continue">
-        <h4 slot="headline" className="usa-alert-heading">
-          Please verify your identity before continuing to My VA Health
-        </h4>
-        <p>
-          We take your privacy seriously, and we’re committed to protecting your
-          information. You’ll need to verify your identity before we can give
-          you access to your personal health information.
-        </p>
-        <button
-          onClick={() => handleClick()}
-          className="usa-button-primary va-button-primary"
-        >
-          Verify your identity
-        </button>
-      </va-alert>
-    </div>
+    <section className="login">
```

Comment:
Similar to above, but reversed. Manually added the login/container elements as this is a unique page that does not include the `LoginContainer`. Since they were removed from application entry, needed to add here